baselines/jft/active_learning_test.py

```diff
@@ -69,15 +69,24 @@ def test_active_learning_script(self):
     config.train_split = 'train[:98%]'
     config.batch_size = 8
     config.max_labels = 3
-    config.acquisition_batch_size = 2
+    config.acquisition_batch_size = 5
+    config.total_steps = 6
     config.dataset_dir = data_dir
     config.model_init = checkpoint_path
+    acquisition_method = 'uniform'
```

Comment:
Can you test all of the new acquisition functions? Best option would be to use same kind of `@parameterized.parameters` setup that [deterministic_test.py](https://github.com/google/uncertainty-baselines/blob/main/baselines/jft/deterministic_test.py#L44) uses, with two parameters for this `test_active_learning_script` function: `acquisition_method` and `correct_ids`. One line for each of the four new acquisition methods.

Reply:
I tried to do this, but I couldn't figure out how to do the test.

The first acquisition is always done with `uniform` so to test the others we would need to first acquire with `uniform`, then train, then acquire with the other methods. Unfortunately the train step always crashes since we filter by id, but the ids are randomized (by `tfds.testing.mock_data`) so we end up with an empty training set.